plugins-builtin/extractcode-libarchive-system-provided/README.md

```diff
@@ -0,0 +1 @@
+A scanCode debian package clubbed plugin to provide pre-built binary libraries,utilities and their locations.
```

Comment:
scanCode -> ScanCode
Also is this specific to Debian?

Reply:
Ack! Correctly pointed out these plugins are to be used with fedora package or other distribution package as well if needed.